packages/klevu-ui/src/components/klevu-slider/klevu-slider.tsx

```diff
@@ -81,6 +164,7 @@ export class KlevuSlider {
     el.noUiSlider.on("end", (ev) => {
       this.klevuSliderChange.emit([parseFloat(ev[0]), parseFloat(ev[1])])
     })
+    mergeTooltips(el, 40, " - ")
```

Comment:
```suggestion
    mergeTooltips(el, 20, " - ")
```

Tested the slider. 40 is quite big number. With 20 we get better looking result.